gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/persistence/PersistenceAdvisorImpl.java

```diff
@@ -406,7 +412,8 @@ public void setInitializing(PersistentMemberID newId) {
       //will remove that ID from the peers.
       if(initializingId != null) {
         if(traceOn()) {
-          trace(" We still have an initializing id: " + initializingId + "  Telling peers to remove the old id " + oldId + " and transitioning this initializing id to old id. recipients " + profileUpdateRecipients);
+          trace(" We still have an initializing id: " + initializingId + "  Telling peers to remove the old id " +
```

Comment:
Please correct the line length. We are following the max line length < 100 for all source files irrespective of the types.